src/pages/sangminLee/Main/Main.js

```diff
@@ -1,8 +1,201 @@
+              <li className="side-story">
+                <div className="story-profile" />
+                <div className="story-title">
+                  <div className="story-name">ggomi5</div>
+                  <div className="story-time">11분 전</div>
+                </div>
+              </li>
+            </ul>
+          </div>
+          {/* 회원님을 위한 추천 */}
+          <div className="side-recommandsbox">
+            <div className="side-recommandstitle">
+              <span className="blur">회원님을 위한 추천</span>{' '}
+              <span>모두 보기</span>
+            </div>
+            <ul className="side-recommandslist">
+              <li className="side-recommand">
+                <div className="recommand-profile" />
+                <div className="recommand-title">
+                  <div className="recommand-name">ggomi6</div>
+                  <div className="recommand-follower">
+                    회원님을 팔로우합니다
+                  </div>
+                </div>
+                <button>팔로우</button>
+              </li>
+              <li className="side-recommand">
+                <div className="recommand-profile" />
+                <div className="recommand-title">
+                  <div className="recommand-name">ggomi7</div>
+                  <div className="recommand-follower">
+                    회원님을 팔로우합니다
+                  </div>
+                </div>
+                <button>팔로우</button>
+              </li>
+              <li className="side-recommand">
+                <div className="recommand-profile" />
+                <div className="recommand-title">
+                  <div className="recommand-name">ggomi8</div>
+                  <div className="recommand-follower">
+                    회원님을 팔로우합니다
+                  </div>
+                </div>
+                <button>팔로우</button>
+              </li>
+              <li className="side-recommand">
+                <div className="recommand-profile" />
+                <div className="recommand-title">
+                  <div className="recommand-name">ggomi9</div>
+                  <div className="recommand-follower">
+                    회원님을 팔로우합니다
+                  </div>
+                </div>
+                <button>팔로우</button>
+              </li>
+              <li className="side-recommand">
+                <div className="recommand-profile" />
+                <div className="recommand-title">
+                  <div className="recommand-name">ggomi10</div>
+                  <div className="recommand-follower">
+                    회원님을 팔로우합니다
+                  </div>
+                </div>
+                <button>팔로우</button>
+              </li>
+            </ul>
+          </div>
+
+          <div className="side-information">
+            Westagram · 정보 · 지원 · 홍보센터 · API · 채용 · 정보 ·
+            개인정보처리방침 · 약관 · 디렉터리 · 프로필 · 해시태그 · 언어
+          </div>
+          <div className="side-company">© 2022 Westagram</div>
```

Comment:
Footer 컴포넌트로 분리해주고,div 대신 footer 시멘틱 태그를 사용해보면 좋을 듯해요!
```suggestion
          <Footer />
```

Reply:
각 메뉴 name을 JS 파일에 저장하고,
map 메소드로 출력해주는 방식으로 수정해보면 좋을 듯해요